mne/viz/epochs.py

```diff
@@ -65,74 +78,290 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
-        axes are passed an error is raised. Defaults to None.
+        axes are passed, an error is raised.
+        If `groupby` is a dict, this cannot be a list, but it can be a dict of
+        lists of axes, with the keys matching those of `groupby`. In that case,
+        the provided axes will be used for the corresponding groups.
+        Defaults to None.
     overlay_times : array-like, shape (n_epochs,) | None
         If not None the parameter is interpreted as time instants in seconds
         and is added to the image. It is typically useful to display reaction
         times. Note that it is defined with respect to the order
         of epochs such that overlay_times[0] corresponds to epochs[0].
+    combine : None | str | callable
+        If None, return one figure per pick. If not None, aggregate over
+        channels via the indicated method. If str, must be one of "mean",
+        "median", "std" or "gfp", in which case the mean, the median, the
+        standard deviation or the GFP over channels are plotted.
+        array (n_epochs, n_times).
+        If callable, it must accept one positional input, the data
+        in the format (n_epochs, n_channels, n_times). It must return an
+        array (n_epochs, n_times). For example:
+
+            `combine = lambda data: np.median(data, 1)`
+
+        Defaults to None if picks are given, otherwise 'gfp'.
+
+    groupby : None | str | dict
+        If not None, combining happens over channel groups defined by this
+        parameter.
+        If str, must be "type", in which case one figure per channel type is
+        returned (combining within channel types).
+        If a dict, the values must be picks and one figure will be returned
+        for each entry, aggregating over the corresponding pick groups; keys
+        will become plot titles. This is useful for e.g. ROIs. Each entry must
+        contain only one channel type. For example:
+
+           `groupby=dict(Left_ROI=[1, 2, 3, 4], Right_ROI=[5, 6, 7, 8])`
+
+        If not None, combine must not be None. Defaults to None if picks are
+        given, otherwise 'type'.
+
+    evoked : Bool
+        Draw the ER[P/F] below the image or not.
+    ts_args : dict
+        Arguments passed to a call to `mne.viz.plot_compare_evoked` to style
+        the evoked plot below the image. Defaults to an empty dictionary,
+        meaning `plot_compare_evokeds` will be called with default parameters.
 
     Returns
     -------
     figs : lists of matplotlib figures
         One figure per channel displayed.
     """
-    from scipy import ndimage
     units = _handle_default('units', units)
     scalings = _handle_default('scalings', scalings)
 
-    import matplotlib.pyplot as plt
+    # setting defaults
+    if groupby is not None and combine is None:
+        combine = 'gfp'
+
+    if all(param is None for param in (groupby, picks, combine)):
+        groupby = "type"
+        combine = "gfp"
```

Comment:
Oh I see. Maybe add a comment about this in the ``group_by`` and ``combine`` docs.